GIBSDownloader/tiff_downloader.py

```diff
@@ -5,24 +5,25 @@
 from GIBSDownloader.coordinate_utils import Rectangle, Coordinate
 
 class TiffDownloader():
+
+    @classmethod
+    def generate_download_filename(cls, output, product, date):
+        return "{}{}_{}".format(output, str(product), date)
+
     @classmethod
-    def download_area_tiff(cls, region, date, xml_path, output, product, width=None, height=None, out_frmt="GTiff"):
-        """
-        region: rectangular region to be downloaded
-        date: YYYY-MM-DD
-        output: path/to/filename (do not specify extension)
-        returns tuple with dowloaded width and height
-        """
+    def download_area_tiff(cls, region, date, xml_path, filename, product, width=None, height=None, out_frmt="GTiff"):
```

Comment:
You removed the doc here. Was that intentional?

Reply:
Yeah, those were old docs that no longer matched the function and I wanted to keep consistent with the rest of the package